libappimage/CMakeLists.txt

```diff
@@ -1,83 +1,78 @@
-project(appimagelib)
+project(AppImage)
```

Comment:
Please call it libappimage, the AppImage project is the top level one (although it lacks that line as of now).